roles/pulp_workers/templates/pulpcore-worker@.service.j2

```diff
@@ -4,8 +4,8 @@ After=network-online.target
 Wants=network-online.target
 
 [Service]
-EnvironmentFile=-/etc/default/pulp-workers
-EnvironmentFile=-/etc/default/pulp-workers-%i
+EnvironmentFile=-/etc/default/pulp_workers
```

Comment:
Do we want to rename these?

Reply:
Not sure, I just changed for consistency

Reply:
I'm not sure what we should do.

If users do have any environment files already, it's another file they have to rename.

Also, the /etc/default/ file should probably be named after the systemd unit name. So it should be /etc/default/pulpcore-worker or /etc/default/pulpcore-workers .

@pulp/ansible-installer what do you guys think?

Reply:
I agree with you @mikedep333 the goal of this PR is to change the role names, so I think we don't need to change file names, and setting it back to have dash